.buildkite/pipeline.yml

```diff
@@ -0,0 +1,26 @@
+##################################
+# Docker image deployment pipeline
+##################################
+steps:
+  - label: Get docker tag and save it as metadata for use later
+    command: .buildkite/scripts/set_docker_tag_meta_data.sh
+
+  - wait
+
+  - label: Build docker image (master branches only)
+#    branches: master
```

Comment:
Do we need to uncomment this before merging?

Reply:
Yes!

Reply:
Replaced with `branches: "*"`